packages/docs/src/page-configs/ui-elements/select/page-config.ts

```diff
@@ -80,6 +80,11 @@ const config: ApiDocsBlock[] = [
     'select.examples.selectedTopShown.text',
     'SelectedTopShown',
   ),
+  ...block.exampleBlock(
+    'select.examples.autocomplete.title',
+    'select.examples.autocomplete.text',
+    'Autocomplete',
```

Comment:
I suggest we add an example with item create. That's a very common case.

Also a demo with relevant event.